packages/sync/src/class-actions.php

```diff
@@ -691,14 +691,13 @@ public static function get_start_time_offset( $schedule = '', $hook = '' ) {
 		 * @param string $hook
 		 * @param string $schedule
 		 */
-		return intval(
+		return (int)
 			apply_filters(
 				'jetpack_sync_cron_start_time_offset',
 				$start_time_offset,
 				$hook,
 				$schedule
```

Comment:
Indenting seems to be off on this one:

```suggestion
		return (int) apply_filters(
			'jetpack_sync_cron_start_time_offset',
			$start_time_offset,
			$hook,
			$schedule
```